### DISCUSSION TITLE
Mock pump not working

### DISCUSSION THREAD

**Discussion by kevincw01** [tagyoureit/nodejs-poolController#1045] [Nixie]
I have a nixie single body working with a pentair pump correctly and I want to make some more code changes without disrupting what works.  I changed my rs485 connection to "mock" in the config.json but I don't see any successful receives.  I do see a message in the logs about a successful mock port open.  Any tips to get this working?

**Comment by tagyoureit** [comment]
The mock port works for most of the controller commands.  It hasn't been extended as far as mocking pumps or other hardware.
